src/components/header/headerNavbar.tsx

```diff
@@ -15,6 +15,7 @@ const HeaderNavbar = () => {
         <Navbar.Collapse id="responsive-navbar-nav">
           <Nav className="justify-content-center flex-fill" style={{height: '36px'}}>
             <Nav.Link href="/" className="mx-2 p-0"><Button variant="secondary" className="p-0 w-100 h-100 m-0 px-2 text-white">Home</Button></Nav.Link>
+            <Nav.Link href="/browse" className="mx-2 p-0"><Button variant="primary" className="p-0 w-100 h-100 m-0 px-2 text-white">Browse</Button></Nav.Link>
```

Comment:
On click, the browse button needs to change color to orange. The current page will be highlighted using the orange color.

Reply:
We should make this its own PR. It's a big task.

Reply:
Not so big in the end, but done in this PR as discussed.